Makefile

```diff
@@ -103,6 +103,12 @@ test:
 	set -e; python3 -m compileall -q -x '/vmware-tools/scripts/, /ppp/' .
 	PYTHONPATH=python/ python3 -m "nose" --with-xunit src --with-coverage --cover-erase --cover-xml --cover-package src/conf_mode,src/op_mode,src/completion,src/helpers,src/validators,src/tests --verbose
 
+.PHONY: migration_scripts
```

Comment:
Can you please rename this to: `check_migration_scripts_executable`? Other then that it looks good to me.

Reply:
Done